e2e/internal/e2e/docker.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package e2e
+
+import (
+	"os/exec"
+	"strings"
+	"testing"
+)
+
+func PrepRegistry(t *testing.T) {
+	commands := [][]string{
```

Comment:
I did not look at all the details about the E2E framework but do we need to call `test.EnsurePrivilege(t)` or `test.DropPrivilege(t)` and `defer test.ResetPrivilege(t)`?

Reply:
@gvallee I do not think so as the `e2e` framework is always ran as root